Library/SettingsAccountCellType.swift

```diff
@@ -75,12 +84,14 @@ public enum SettingsAccountCellType: SettingsCellTypeProtocol {
       return Strings.Payment_methods()
     case .currency:
       return Strings.Currency()
+    case .currencyPicker:
+     return ""
     }
   }
 
   public var description: String? {
     switch self {
-    case .currency: return "$ Dollar (USD)"
+    case .currency: return Strings.Currency_USD()
```

Comment:
I think we can remove this since we're reading the currency from the `SettingsCurrencyCellValue` now

Reply:
Can we remove this case?